### PR TITLE
Feature/gasprices

### PR DESCRIPTION
**Pull request by jose-donato**
# Description

- [ ] Summary of the change / bug fix.
- [ ] Link # issue, if applicable.
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
- [ ] Relevant motivation and context.
- [ ] List any dependencies that are required for this change.

# How has this been tested?

* Please describe the tests that you ran to verify your changes.
* Provide instructions so we can reproduce.
* Please also list any relevant details for your test configuration.
- [ ] Make sure affected commands still run in terminal
- [ ] Ensure the SDK still works
- [ ] Check any related reports

# Checklist:

- [ ] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [ ] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [ ] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [ ] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

File: openbb_terminal/alternative/collectapi_model.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    Parameters
+    ----------
+
+    Returns
+    -------
+    pd.DataFrame
+        DataFrame with data
+    """
+    res = requests.get(
```
Comment (jmaslek) [inline openbb_terminal/alternative/collectapi_model.py:27]: Can you add a timeout?  I want to start enforcing this across.

File: openbb_terminal/config_terminal.py [hunk 2]
```diff
@@ -192,3 +192,6 @@
 
 # https://stocksera.pythonanywhere.com/accounts/developers
 API_STOCKSERA_KEY = os.getenv("OPENBB_API_STOCKSERA_KEY") or "REPLACE_ME"
+
+# https://docs.collectapi.com/docs/general
+API_COLLECTAPI_KEY = os.getenv("OPENBB_API_COLLECTAPI_KEY") or "REPLACE_ME"
```
Comment (jmaslek) [inline openbb_terminal/config_terminal.py:197]: needs to be added to the key menu.

**Comment by jmaslek** [comment]
*cough* unit tests *cough*

**Review by jmaslek — CHANGES_REQUESTED** [review]

**Comment by jose-donato** [comment]
Apparently their free trial is only few days.. 😞

**Comment by jmaslek** [comment]
jose said i could close this

**Pull request closed without merging**